README.md

```diff
@@ -152,7 +220,6 @@ IndexModel indexModel = pinecone.createPodsIndex(indexName, dimension, environme
 
 
 ## List indexes
-
 The following example returns all indexes (and their corresponding metadata) in your project.
```

Comment:
**nit:** I think it's generally considered best practice leave line breaks on either side of headings in markdown. My IDE has some formatting that automatically applies these when I'm working in these files: https://www.markdownguide.org/basic-syntax/#heading-best-practices

Reply:
Ahh interesting, let me add new lines